ssl/quic/quic_port.c

```diff
@@ -136,6 +140,12 @@ static int port_init(QUIC_PORT *port)
     ossl_list_port_insert_tail(&port->engine->port_list, port);
     port->on_engine_list    = 1;
     port->bio_changed       = 1;
+
+    /* Generate random key for token encryption */
+    if (!RAND_bytes_ex(port->engine->libctx, port->token_key,
+        sizeof(port->token_key), 0))
```

Comment:
different ciphers have different key lengths.  Might be good to replace the token_key array with a pointer and allocate it based on the value returned from EVP_CIPHER_get_key_length.  Stinks that we then have to manage the allocation and freeing, but its likely safer for future proofing

Reply:
also, its an irritating style nit, but the style checker says the sizeof should line up with the port variable in the call to RAND_bytes_ex